### ISSUE TITLE
v1.12.0 update errors on c# Targets: Error processing 'configurationDone' request. Unknown Error: 0x89720010

### ISSUE THREAD

**Issue by jasonsturges** [Microsoft/vscode#25918]
- VSCode Version: 1.12.0 
   - 72672be0b7d3eef0784077b880615f91b7ec85aa
   - 2017-05-04T16:07:40.729Z
- OS Version: OS X El Capitan (10.11.6)

With today's release of VS Code 1.12.0, debugging my ASP.NET Core now gives error: 

> Error processing 'configurationDone' request. Unknown Error: 0x89720010

![image](https://cloud.githubusercontent.com/assets/1213591/25719353/c27ed452-30ce-11e7-8e09-3002a6335b37.png)

No issues up until this update.

Steps to Reproduce:

1. Update to VS Code 1.12.0

2. Debug application by pressing green arrow.

This occurs with every ASP.NET Core web application or DotNet Core command line application I've created (all of which previously debugged without issue).

**Comment by kieferrm** [comment]
@jasonsturges you are using the csharp extension?

**Comment by kieferrm** [comment]
Duplicate of https://github.com/OmniSharp/omnisharp-vscode/issues/1454.

//cc @weinand @isidorn @roblourens

**Comment by jasonsturges** [comment]
@kieferrm Yes - ms-vscode.csharp, C# for Visual Studio Code (powered by OmniSharp).

I guess that extension is not compatible with this update to Visual Studio Code.

**Comment by kieferrm** [comment]
@jasonsturges we pinged the extension folks. We'll see what they come back with.

**Comment by kieferrm** [comment]
@jasonsturges until you are unblocked here is a [link to 1.11.2](https://az764295.vo.msecnd.net/stable/6eaebe3b9c70406d67c97779468c324a7a95db0e/VSCode-darwin-stable.zip)

**Comment by BlakeWills** [comment]
@kieferrm How can I install this on Ubuntu?

**Comment by Tyriar** [comment]
@BlakeWills https://packages.microsoft.com/repos/vscode/pool/main/c/code/code_1.11.2-1492070517_amd64.deb

**Comment by gregg-miskelly** [comment]
@kieferrm you likely want to reopen this issue since this should be tracked here.

**Comment by kieferrm** [comment]
Fixed with ef63fc88b3c0a9102966c6f82f7e2096d1b37a16.

**Comment by kieferrm** [comment]
In case you wonder: Between the last electron update and this one we had added a step to our stable build process that needs to be performed manual. It's the only step that's manual and usually needs to be performed once during an iteration. Guess which step we missed? Building is all about automation....

**Comment by awerlang** [comment]
Sorry for highjacking this issue, but what if I don't care about an update, can I inform VS Code that I definitely don't want to be bothered with that one? I've updated to 1.12 and don't need the fix. Bugs always find their way in, but I may not be interested at all in a fix. And I know there would be other times as well, like localization releases. This issue is kind of the trigger for it, let me know if you want me to open an issue.

Thanks!

**Comment by weinand** [comment]
@awerlang you can set your user preference `update.channel` to `none` if you do not want to be bothered by new releases.
Please be aware that we only push recovery releases for critical issues that affect many users (e.g. data loss). We do not push new localizations this way.

**Comment by awerlang** [comment]
Thanks Weinand!

**Comment by LukeMarlin** [comment]
Getting the issue, not sure how to troubleshoot.
Tried 2 versions: 
Version 1.18.0 (Commit 02b9fb1fd9ca8239986cade543b5e0e1a841834c)
and right now I'm testing Version 1.17.2 (Commit b813d12980308015bcd2b3a2f6efa5c810c33ba5)
